src/react_native/fast_image.cljs

```diff
@@ -1,40 +1,70 @@
 (ns react-native.fast-image
   (:require
     ["react-native-fast-image" :as FastImage]
+    [clojure.string :as string]
+    [oops.core :as oops]
     [react-native.core :as rn]
     [reagent.core :as reagent]))
 
-(def fast-image-class (reagent/adapt-react-class ^js FastImage))
+(defn- build-source
+  [source]
+  (if (string? source)
+    {:uri      source
+     :priority :high}
+    source))
 
-(defn placeholder
-  [style child]
-  [rn/view {:style (merge style {:flex 1 :justify-content :center :align-items :center})}
-   child])
+(defn- remove-port
+  [source]
+  (if (string? source)
+    (string/replace-first source #":\d+" "")
+    (string/replace-first (oops/oget source :uri) #":\d+" "")))
```

Comment:
I'm not sure, but does `oops/get` throw an exception if it doesn't find `:uri`? If so, it might be worth using `goog.object/get` with a default value like: `(goog.object/get source :uri "")`

Reply:
Good question @seanstrom. Some time ago I changed the [default behavior of oops during development](https://github.com/binaryage/cljs-oops?tab=readme-ov-file#play-it-safe-during-development), which is to instrument calls and throw on missing keys. Now it just warns. In advanced compilation instrumentation in oops is disabled by default and we don't have to configure anything special.

https://github.com/status-im/status-mobile/blob/eae887e6b49974547c944c0ec3203dba1f01e8f1/src/status_im/setup/oops.cljs#L5-L12